app/models/sub_service_request.rb

```diff
@@ -264,14 +264,41 @@ def ctrc?
     self.organization.tag_list.include? "ctrc"
   end
 
+  # Can't edit a Nexus ssr if it's placed in an uneditable status
   def can_be_edited?
-    ['first_draft', 'draft', 'submitted', nil, 'obtain_research_pricing'].include?(self.status) ? true : false
```

Comment:
Yes – no matter the status, all other requests can be edited at any point.

From: amcates [mailto:notifications@github.com]
Sent: Thursday, February 12, 2015 10:09 AM
To: sparc-request/sparc-request
Subject: Re: [sparc-request] Restricting editing of requests by status (#36)

CAUTION: External

In app/models/sub_service_request.rb<https://github.com/sparc-request/sparc-request/pull/36#discussion_r24586906>:

>    def can_be_edited?

> -    ['first_draft', 'draft', 'submitted', nil, 'obtain_research_pricing'].include?(self.status) ? true : false

So, does this mean that all other requests can be edited regardless of status.

—
Reply to this email directly or view it on GitHub<https://github.com/sparc-request/sparc-request/pull/36/files#r24586906>.